From ee56a688a46a2abc2ca46c061898b27b44fb9602 Mon Sep 17 00:00:00 2001
From: Leon Anavi <leon.anavi@konsulko.com>
Date: Thu, 17 Jul 2025 16:23:58 +0300
Subject: [PATCH] pyproject.toml: Update mypy requirement

Updates the requirements on mypy to permit the latest version.

Upstream-Status: Submitted [https://github.com/jawah/charset_normalizer/pull/632]

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index e7f052a..3a75dc1 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm", "mypy>=1.4.1,<=1.15.0"]
+requires = ["setuptools", "setuptools-scm", "mypy>=1.4.1,<=1.17.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
-- 
2.39.5

